source/documentation/other-topics/migrate-to-live.html.md.erb

```diff
@@ -266,6 +266,13 @@ Earlier on in this migration you will have [migrated the ServiceAccount module](
 
 An example of GitHub Actions config is provided in this repo: https://github.com/ministryofjustice/cloud-platform-reference-app-github-action. It deploys the app to both the live-1 and live clusters.
 
+### Namespace dependencies
```

Comment:
I wonder if there is a more logical place to put this info, than mixed into the step about changing their app deployment pipeline? Can it be a prerequisite, for example? Or a separate step?

Reply:
Added as a seperate step (Step 7).